src/App.tsx

```diff
@@ -0,0 +1,14 @@
+import { useState } from 'react'
+import './App.css'
+
+function App() {
+  
+
```

Comment:
[NITPICK] please use some linter to format your code. For now you can use prettier for linting.